crates/rv/src/commands/ruby/install.rs

```diff
@@ -40,30 +39,25 @@ type Result<T> = miette::Result<T, Error>;
 pub async fn install(
     config: &Config,
     install_dir: Option<String>,
-    requested: &RubyRequest,
+    version: Option<&RubyRequest>,
     tarball_path: Option<String>,
 ) -> Result<()> {
-    let install_dir = match install_dir {
-        Some(dir) => Utf8PathBuf::from(dir),
-        None => match config.ruby_dirs.first() {
-            Some(dir) => dir.clone(),
-            None => panic!("No Ruby directories to install into"),
-        },
-    };
+    let version = version.unwrap_or_else(||
+        config.ruby_request().expect("Couldn't find a .ruby-version or .tool-versions").as_ref()
+    );
```

Comment:
Now I'm getting `cannot return value referencing temporary value; returns a value referencing data owned by the current function (rustc E0515)` here, but calling `clone()` doesn't seem to be the fix.